### PR TITLE
Python flask CD template

### PR DESCRIPTION
**Pull request by chperich**

### CODE REVIEW

File: templates/python-package-to-windows-webapp-on-azure.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+              WebAppName: $(webAppName)
+              ExtensionsList: python353x86
+              OutputVariable: 'PYTHON_EXT'
+
+          - task: AzureRmWebAppDeployment@4
+            displayName: 'Azure App Service Deploy'
+            inputs:
+              azureSubscription: $(azureServiceConnectionId)
+              WebAppName: $(webAppName)
+              packageForLinux: '$(Pipeline.Workspace)/drop/$(Build.BuildId).zip'
```
Comment (vincentdass) [inline templates/python-package-to-windows-webapp-on-azure.yml]: can u check this? I think we removed this input

File: templates/python-package-to-windows-webapp-on-azure.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        deploy:
+          steps:
+          - task: AzureAppServiceManage@0
+            displayName: 'Install Python Extension'
+            inputs:
+              azureSubscription: $(azureServiceConnectionId)
+              Action: 'Install Extensions'
+              WebAppName: $(webAppName)
+              ExtensionsList: python353x86
+              OutputVariable: 'PYTHON_EXT'
```
Comment (vincentdass) [inline templates/python-package-to-windows-webapp-on-azure.yml]: this input is disable I gues,, can u check this variable? `ExtensionsList: '$(D)`

File: templates/python-package-to-windows-webapp-on-azure.yml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+                )
+                call "$(PYTHON_EXT)/python.exe" -m pip install -r requirements.txt
+                if %errorlevel% NEQ 0 (
+                 echo Failed to install dependencies>&2
+                 EXIT /b 1
+                )
+              WebConfigParameters: '-WSGI_HANDLER "$(appFolderName).app"  -PYTHON_PATH "$(PYTHON_EXT)\python.exe"  -PYTHON_WFASTCGI_PATH "$(PYTHON_EXT)\wfastcgi.py" -STATIC_FOLDER_PATH "$(appFolderName)\static" -appType python_Flask'
+              enableCustomDeployment: true
+              ExcludeFilesFromAppDataFlag: false
+              DeploymentType: webDeploy
```
Comment (vincentdass) [inline templates/python-package-to-windows-webapp-on-azure.yml]: can we switch to `Run From Package` and observe the behaviour?

File: templates/python-package-to-windows-webapp-on-azure.yml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  azureServiceConnectionId: '{{ azureServiceConnection.Id }}'
+  
+  # Web app name
+  webAppName: '{{ webAppName }}'
+
+  # Agent VM image name
+  vmImageName: 'windows-latest'
+
+  # Environment name
+  environmentName: '{{ webAppName }}'
```
Comment (ds-ms) [inline templates/python-package-to-windows-webapp-on-azure.yml]: Given that it's only used once, perhaps you can put the `{{ webAppName }}` directly?

File: templates/python-package-to-windows-webapp-on-azure.yml [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+  # Project root folder. Point to the folder containing manage.py file.
+  projectRoot: $(System.DefaultWorkingDirectory)
```
Comment (ds-ms) [inline templates/python-package-to-windows-webapp-on-azure.yml]: Same here

File: templates/python-package-to-windows-webapp-on-azure.yml (hunk 1 above)
Comment (chandanapericharla) [reply templates/python-package-to-windows-webapp-on-azure.yml]: Changed it to package.

File: templates/python-package-to-windows-webapp-on-azure.yml (hunk 2 above)
Comment (chandanapericharla) [reply templates/python-package-to-windows-webapp-on-azure.yml]: As discussed offline. Continuing to use it.

File: templates/python-package-to-windows-webapp-on-azure.yml (hunk 3 above)
Comment (chandanapericharla) [reply templates/python-package-to-windows-webapp-on-azure.yml]: Changed it to zip deploy.

File: templates/python-package-to-windows-webapp-on-azure.yml (hunk 4 above)
Comment (chandanapericharla) [reply templates/python-package-to-windows-webapp-on-azure.yml]: This is just some default value we are trying to populate for the user. We expect this to be edited by the user to suite their needs. Making it a variable will make it easy for them to find it and edit. Would not want to hide it deep in the script.

File: templates/python-package-to-windows-webapp-on-azure.yml (hunk 5 above)
Comment (chandanapericharla) [reply templates/python-package-to-windows-webapp-on-azure.yml]: Same here.

**Comment by chandanapericharla** [comment]
> change to zip deploy

Done

File: templates/python-package-to-windows-webapp-on-azure.yml (hunk 4 above)
Comment (ds-ms) [reply templates/python-package-to-windows-webapp-on-azure.yml]: That won't be the case right, we would have already created an environment with that name by then?

**Pull request merged**